### PR TITLE
carousel data from attachements

### PR DESCRIPTION
**Pull request by randyhoulahan**

### CODE REVIEW

**Comment by blaisefonseca** [comment]
you will need somthing to bust the cache

**Comment by randyhoulahan** [comment]
updated

**Comment by randyhoulahan** [comment]
@sbilodeau SVP

File: app/directives/bbi/slider.html [hunk 1]
```diff
@@ -101,7 +101,7 @@ <h5 class="main-title"><img src="/app/images/bbi/logo_white.svg" style="height:7
         <div class="carousel-inner" role="listbox" style="height:400px;">
 
             <div class="item pointer" ng-repeat="item in carouselData.items" ng-class="{active: $index==0}">
-                <div class="img-cont" ng-click="goTo(item.targetUrl)" >
+                <div class="img-cont" ng-click="go(item.targetUrl)" >
```
Comment (sbilodeau) [inline app/directives/bbi/slider.html]: whay not using <a> + target=_blank

Comment (randyhoulahan) [reply app/directives/bbi/slider.html]: sometimes the the link is to a page within the SPA

File: app/directives/bbi/slider.js [hunk 2]
```diff
@@ -103,14 +76,16 @@ define(['app', 'text!./slider.html','lodash','directives/bbi/ani-scroll'], funct
...
             controller: function ($scope, $location) {
-                //============================================================
-                //
-                //============================================================
-                $scope.goTo = function (url) {
-
-                        $location.path(url);
+              $scope.go = function (url) {
+                try {
+                  (new URL(url)).hostname
```
Comment (sbilodeau) [inline app/directives/bbi/slider.js]: what is this line?

Comment (randyhoulahan) [reply app/directives/bbi/slider.js]: test for valid url

Comment (sbilodeau) [reply app/directives/bbi/slider.js]: what is the purpose?

**Pull request merged**